python/lsst/obs/hsc/hscMapper.py

```diff
@@ -230,11 +226,11 @@ def map_linearizer(self, dataId, write=False):
             storage=self.rootStorage)
 
     def bypass_linearizer(self, datasetType, pythonType, butlerLocation, dataId):
-        """Return the linearizer.
+        """Return a Linearizer() for a particular detector. Linearizer() applies to
+        one detector only (i.e., it is detector-specific). When processing multiple
+        detectors, a new instance of Lineaizer() will be created each time.
```

Comment:
Could you please make this docstring follow the [numpydoc guidelines](https://developer.lsst.io/python/numpydoc.html#py-docstring-sections)?

It would also be helpful to include a little more information about the `Linearizer` that is being returned.

Specifically, I'd suggest something like:

```python
"""Return a linearizer for the given detector.

On each call, a fresh instance of `Linearizer` is returned; the caller is responsible for initializing it
appropriately for the detector.

Parameters
----------
`datasetType`: ...
`pythonType`: ...
`butlerLocation`: ...
`dataId`: ...

Notes
-----
Linearizers are not saved to persistent storage; rather, they are managed entirely in memory.
On each call, this function will return a new instance of `Linearizer`, which must be managed
(including setting it up for use with a particular detector) by the caller. Calling
`bypass_linearizer` twice for the same detector will return _different_ instances of `Linearizer`,
which share no state.
"""
```